percona-packaging/scripts/psmdb_builder.sh

```diff
@@ -794,9 +926,9 @@ DEB_RELEASE=1
 REVISION=0
 BRANCH="master"
 REPO="https://github.com/percona/percona-server-mongodb.git"
-PSM_VER="4.0.9"
+PSM_VER="4.4.0"
 PSM_RELEASE="1"
-MONGO_TOOLS_TAG="r4.0.9"
+MONGO_TOOLS_TAG="master"
```

Comment:
are you sure about master tag?

Reply:
As for now we can leave "master" there, since https://github.com/mongodb/mongo-tools.git doesn't have 4.4.0 branch/tag.

Reply:
ok